Assets/Scripts/UI/InGameUI/ConstructionMenu.cs

```diff
@@ -22,6 +22,18 @@ public class ConstructionMenu : MonoBehaviour
 
     private BuildModeController bmc;
 
+    private GameObject[] FurnitureSubs 
+    {
+        get 
+        {
+            return new GameObject[] 
+            {
+                // add every furniture submenu here
+                furnitureMenu, floorMenu
+            };
+        }
+    }
+
```

Comment:
I think its better to just have it as an auto property and set it whenever `furnitureMenu` or `floorMenu` is set:

    private GameObject[] FurnitureSubs { get; private set; }